pkg/engine/engine.go

```diff
@@ -92,6 +97,20 @@ type Engine struct {
 
 	// verify determines whether the scanner will attempt to verify candidate secrets
 	verify bool
+
+	// Note: bad hack only used for testing
+	reverificationTracking *reverificationTracking
```

Comment:
Nit: consider if we should rename this and other related variables to be more close to the flag name